src/features/click-to-play.js

```diff
@@ -2099,11 +2182,7 @@ async function createYouTubePreview (originalElement, widget) {
     youTubePreview.id = `yt-ctl-preview-${widget.widgetID}`
     youTubePreview.style.cssText = styles.wrapperDiv + styles.placeholderWrapperDiv
 
-    // Put our custom font-faces inside the wrapper element, since
-    // @font-face does not work inside a shadowRoot.
-    // See https://github.com/mdn/interactive-examples/issues/887.
-    const fontFaceStyleElement = document.createElement('style')
-    fontFaceStyleElement.textContent = styles.fontStyle
+    const fontFaceStyleElement = makeFontFaceStyleElement()
```

Comment:
Nit: Might as well skip the intermediary variable?
```suggestion 
   youTubePreview.appendChild(makeFontFaceStyleElement())
```